core/src/execution/proto/expr.proto

```diff
@@ -233,12 +233,20 @@ message Remainder {
   DataType return_type = 4;
 }
 
+enum EvalMode {
+  LEGACY = 0;
+  TRY = 1;
+  ANSI = 2;
+}
+
 message Cast {
   Expr child = 1;
   DataType datatype = 2;
   string timezone = 3;
-  // LEGACY, ANSI, or TRY
-  string eval_mode = 4;
+  // Depreciateid: LEGACY, ANSI, or TRY - preserved for backward compatibity
```

Comment:
No need for backwards compatibility here since we have not published a release yet

Reply:
Removed `string eval_mode`.